jest.config.js

```diff
@@ -5,8 +5,8 @@ module.exports = {
 		global: {
 			branches: 66,
 			functions: 100,
-			lines: 100,
-			statements: 100
+			lines: 70,
```

Comment:
I'm not opposed to just removing these thresholds....just saying. 😄

Reply:
@alexplischke and @mhan83 
TBH, I'm a bit against it without knowing the context. We try to always keep the test coverage as high as possible. Can we add some unit tests for this or is this basically not possible